Gemfile

```diff
@@ -7,11 +7,12 @@ gemspec
 group :development do
   gem 'codecov'
   gem 'github_changelog_generator' if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.0')
+  gem 'pry'
   gem 'puppet', ENV['PUPPET_GEM_VERSION'] || ENV['PUPPET_VERSION'] || '~> 4.0'
   gem 'simplecov', '~> 0'
   gem 'simplecov-console'
   if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.1.0')
-    gem 'rubocop', '< 0.50'
+    gem 'rubocop', '= 0.49'
```

Comment:
hard-coding a very specific value risks conflict with other requirements. Why is this specific version necessary?

Reply:
Without the hard requirement there were dependency issues and rubocop warnings I could not fix without pinning to this version as my rubocop was much older.  It is assumed a developer has the latest rubocop but if 0.41 is used, there are tons of warnings.  We can't use 0.50 due to requirement of newer ruby.